scripts/get_requester_pays_status.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+if (( $# < 2 )); then
+  echo "Usage: $0 TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
+  echo "The Terra bucket path can be formatted as gs://fc-XXXXX or fc-XXXXX"
+  echo "i.e you can run ./set_requester_pays.sh on project_id fc-12345"
```

Comment:
change to `./get_requester_pays.sh`

Reply:
also remove the word "on" in the command and add backticks - so I think this line should be 
```echo "i.e you can run `./get_requester_pays.sh project_id fc-12345`"```